lib/saluki-config/src/lib.rs

```diff
@@ -382,20 +423,6 @@ pub struct GenericConfiguration {
 }
 
 impl GenericConfiguration {
-    fn get(&self, key: &str) -> Option<&Value> {
-        match self.inner.value.find_ref(key) {
-            Some(value) => Some(value),
-            None => {
-                // We might have been given a key that uses nested notation -- `foo.bar` -- but is only present in the
-                // environment variables. We specifically don't want to use a different separator in environment
-                // variables to map to nested key separators, so we simply try again here but with all nested key
-                // separators (`.`) replaced with `_`, to match environment variables.
-                let key = key.replace('.', "_");
-                self.inner.value.find_ref(&key)
```

Comment:
Seems like we haven't added this back anywhere else... which would definitely break something things off the top of my head (mostly ADP-specific configuration settings.)

Reply:
Added `get` back. It now returns a type `T` but it looks like we were only ever using it in the typed functions anyways.